elementary/monitor/api/groups/schema.py

```diff
@@ -0,0 +1,13 @@
+from typing import Dict, List, Union
+
+from pydantic import BaseModel
+
+DbtGroupSchema = Dict[str, Union[list, dict]]
+TagsGroupSchema = Dict[str, List[dict]]
+OwnersGroupSchema = Dict[str, List[dict]]
+
+
+class GroupsSchema(BaseModel):
+    dbt: DbtGroupSchema = dict()
```

Comment:
Although it does work with mutable default values, it is not mentioned in pydantic documentation.
I am thinking that maybe it is safer to use `Field(default_factory=dict)` instead, but this is just might be me NIT picking 🙂

Reply:
[Seems fine](https://stackoverflow.com/a/63808835/8182628).